tests/utils/shopper.js

```diff
@@ -188,85 +188,78 @@ export const shopper = {
+			await expect( page ).toFill( `#${ shippingOrBilling }-last_name`, address.last_name );
+			await expect( page ).toFill( `#${ shippingOrBilling }-address_1`, address.shipping_address_1 );
+			await expect( page ).toFill( `#${ shippingOrBilling }-country input`, address.country );
+			await expect( page ).toFill( `#${ shippingOrBilling }-city`, address.city );
+			await expect( page ).toFill( `#${ shippingOrBilling }-state input`, address.state );
+			await expect( page ).toFill( `#${ shippingOrBilling }-postcode`, address.postcode );
 		},
 
+		// prettier-ignore
 		fillBillingDetails: async ( customerBillingDetails ) => {
-			await expect( page ).toFill(
-				'#billing-first_name',
-				customerBillingDetails.firstname
-			);
-			await expect( page ).toFill(
-				'#billing-last_name',
-				customerBillingDetails.lastname
-			);
-			await expect( page ).toFill(
-				'#components-form-token-input-0',
-				customerBillingDetails.country
-			);
-			await expect( page ).toFill(
-				'#billing-address_1',
-				customerBillingDetails.addressfirstline
-			);
-			await expect( page ).toFill(
-				'#billing-address_2',
-				customerBillingDetails.addresssecondline
-			);
-			await expect( page ).toFill(
-				'#billing-city',
-				customerBillingDetails.city
-			);
-			await expect( page ).toFill(
-				'#components-form-token-input-2',
-				customerBillingDetails.state
-			);
-			await expect( page ).toFill(
-				'#billing-postcode',
-				customerBillingDetails.postcode
-			);
-			await expect( page ).toFill(
-				'#phone',
-				customerBillingDetails.phone
-			);
-			await expect( page ).toFill(
-				'#email',
-				customerBillingDetails.email
-			);
+			await expect( page ).toFill( '#billing-first_name', customerBillingDetails.firstname );
+			await expect( page ).toFill( '#billing-last_name', customerBillingDetails.lastname );
+			await expect( page ).toFill( '#billing-country input', customerBillingDetails.country );
+			await expect( page ).toFill( '#billing-address_1', customerBillingDetails.addressfirstline );
+			await expect( page ).toFill( '#billing-address_2', customerBillingDetails.addresssecondline );
+			await expect( page ).toFill( '#billing-city', customerBillingDetails.city );
+			await expect( page ).toFill( '#billing-state input', customerBillingDetails.state );
+			await expect( page ).toFill( '#billing-postcode', customerBillingDetails.postcode );
+			await expect( page ).toFill( '#phone', customerBillingDetails.phone );
+			await expect( page ).toFill( '#email', customerBillingDetails.email );
+		},
+
+		// prettier-ignore
+		fillShippingDetails: async ( customerShippingDetails ) => {
+			await expect( page ).toFill( '#shipping-first_name', customerShippingDetails.firstname );
+			await expect( page ).toFill( '#shipping-last_name', customerShippingDetails.lastname );
+			await expect( page ).toFill( '#shipping-country input', customerShippingDetails.country );
```

Comment:
The same thing here:

This part is causing the test to fail in CI. In the `shopper. verifyShippingDetails` function, we verify if `customerShippingDetails.company` exists on the page, but we didn't fill in the `company` name on the input field in this function. So we should add bellow this line the following code:

```js
await expect( page ).toFill( '#shipping-company', customerShippingDetails.company );
```

Note: The test is successful locally and not in CI because the `company` input is already prefilled with the correct name, but in CI, it seems to be empty by default.

Reply:
Done!